README.md

```diff
@@ -38,6 +38,17 @@ make configure
 make all
 ```
 
+#### Windows
+
+* Install [WSL2](https://docs.microsoft.com/en-us/windows/wsl/wsl2-install)
+* Enable [Docker support  for WSL2](https://docs.docker.com/docker-for-windows/wsl-tech-preview/)
```

Comment:
don't you need to add docker-compose manually ?

Reply:
Nop, just enabling the WSL2 support in the Docker Desktop applications and thats fine